bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.hbase;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hbase.client.Connection;
+
+import com.google.api.client.repackaged.com.google.common.base.Preconditions;
+
+/**
+ * This class provides a simplified mechansim of creating a programmatic Bigtable Connection.
+ */
+public class BigtableConfiguration {
+  private static final String[] CONNECTION_CLASS_NAMES = {
+    "com.google.cloud.bigtable.hbase1_0.BigtableConnection",
+    "com.google.cloud.bigtable.hbase1_1.BigtableConnection",
```

Comment:
Are we prioritizing 1_0 over 1_1? (getConnectionClass returns the first one in order which matches)